changelog/10241.bugfix.md

```diff
@@ -0,0 +1 @@
+[#10241](https://github.com/RasaHQ/rasa/issues/10241): Bump TensorFlow version to 2.6.1.
```

Comment:
Can you also add the version bump to `google-auth` to the changelog?

Reply:
done